console/src/app/components/features/features.component.ts

```diff
@@ -78,140 +98,59 @@ export class FeaturesComponent {
-      }
-      if (this.toggleStates?.oidcTokenExchange?.state !== ToggleState.INHERITED) {
-        req.setOidcTokenExchange(this.toggleStates?.oidcTokenExchange?.state === ToggleState.ENABLED);
-        changed = true;
-      }
-      if (this.toggleStates?.actions?.state !== ToggleState.INHERITED) {
-        req.setActions(this.toggleStates?.actions?.state === ToggleState.ENABLED);
-        changed = true;
-      }
-      if (this.toggleStates?.oidcSingleV1SessionTermination?.state !== ToggleState.INHERITED) {
-        req.setOidcSingleV1SessionTermination(
-          this.toggleStates?.oidcSingleV1SessionTermination?.state === ToggleState.ENABLED,
-        );
-        changed = true;
-      }
-
-      if (changed) {
-        this.featureService
-          .setInstanceFeatures(req)
-          .then(() => {
-            this.toast.showInfo('POLICY.TOAST.SET', true);
-          })
-          .catch((error) => {
-            this.toast.showError(error);
-          });
-      }
+    const req = create(SetInstanceFeaturesRequestSchema, {
+      actions: this.toggleStates?.actions?.state === ToggleState.ENABLED,
+      consoleUseV2UserApi: this.toggleStates?.consoleUseV2UserApi?.state === ToggleState.ENABLED,
+      debugOidcParentError: this.toggleStates?.debugOidcParentError?.state === ToggleState.ENABLED,
+      disableUserTokenEvent: this.toggleStates?.disableUserTokenEvent?.state === ToggleState.ENABLED,
+      enableBackChannelLogout: this.toggleStates?.enableBackChannelLogout?.state === ToggleState.ENABLED,
+      loginDefaultOrg: this.toggleStates?.loginDefaultOrg?.state === ToggleState.ENABLED,
+      oidcLegacyIntrospection: this.toggleStates?.oidcLegacyIntrospection?.state === ToggleState.ENABLED,
+      oidcSingleV1SessionTermination: this.toggleStates?.oidcSingleV1SessionTermination?.state === ToggleState.ENABLED,
+      oidcTokenExchange: this.toggleStates?.oidcTokenExchange?.state === ToggleState.ENABLED,
+      oidcTriggerIntrospectionProjections:
+        this.toggleStates?.oidcTriggerIntrospectionProjections?.state === ToggleState.ENABLED,
+      permissionCheckV2: this.toggleStates?.permissionCheckV2?.state === ToggleState.ENABLED,
+      userSchema: this.toggleStates?.userSchema?.state === ToggleState.ENABLED,
+      webKey: this.toggleStates?.webKey?.state === ToggleState.ENABLED,
     });
+
+    this.featureService
+      .setInstanceFeatures(req)
+      .then(() => {
+        this.toast.showInfo('POLICY.TOAST.SET', true);
+      })
+      .catch((error) => {
+        this.toast.showError(error);
+      });
   }
 
-  private getFeatures(inheritance: boolean) {
-    this.featureService.getInstanceFeatures(inheritance).then((instanceFeaturesResponse) => {
-      this.featureData = instanceFeaturesResponse.toObject();
+  private getFeatures() {
+    this.featureService.getInstanceFeatures().then((instanceFeaturesResponse) => {
+      this.featureData = instanceFeaturesResponse;
+
       console.log(this.featureData);
```

Comment:
do we wanna keep this console.log?

Reply:
I thought about removing it too.. We usually do not log anything so I remove it.